documentation/source/user_section/courses.rst

```diff
@@ -0,0 +1,32 @@
+.. _courses:
+
+SCT Courses
+###########
+
+We organize a **free** in-person SCT course each year after the ISMRM conference. The course provides a guided walkthrough of SCT, with developers and researchers on-hand to provide support and to answer any questions you may have. If you’d like to be added to the mailing list for course announcements, please visit `this link <https://docs.google.com/forms/d/e/1FAIpQLSdvsXwLRj07bx85O2wPM9ut1sC-20dFKdTVe-OckyvkfncV3Q/viewform>`_.
+
+The past courses handouts are listed below:
+
+-  `SCT course (v4.2.1), London, 2020-01-21`_ \| `Video recording`_
+-  `SCT course (v4.0.0), Beijing, 2019-08-02`_ \| `Slides with Chinese translation`_
+-  `SCT course (v4.0.0_beta.4), London, 2019-01-22`_
+-  `SCT course (v3.2.2), Paris, 2018-06-12`_
+-  `SCT course (v3.0.3), Honolulu, 2017-04-28`_
+-  `SCT course (v3.0_beta14), Geneva, 2016-06-28`_
+-  `SCT course (v3.0_beta9), Singapore, 2016-05-13`_
+-  `SCT course (v3.0_beta1), Montreal, 2016-04-19`_
+-  `SCT Hands-on Workshop (v2.0.4), Toronto, 2015-06-15`_
+
+.. _SCT course (v4.2.1), London, 2020-01-21: https://drive.google.com/file/d/1TZireJ6yhV8q7PbKKXyXg7Heov9-YJMu/view?usp=sharing
+.. _Video recording: https://www.youtube.com/watch?v=whbtjYNtHko&feature=youtu.be
+.. _SCT course (v4.0.0), Beijing, 2019-08-02: https://osf.io/arfv7/
+.. _Slides with Chinese translation: https://osf.io/hnmr2/
+.. _SCT course (v4.0.0_beta.4), London, 2019-01-22: https://osf.io/gvs6f/
+.. _SCT course (v3.2.2), Paris, 2018-06-12: https://osf.io/386h7/
+.. _SCT course (v3.0.3), Honolulu, 2017-04-28: https://osf.io/fvnjq/
+.. _SCT course (v3.0_beta14), Geneva, 2016-06-28: https://drive.google.com/file/d/0Bx3A13n3Q_EAOXktWmNVNGRhdUk/view?usp=sharing
+.. _SCT course (v3.0_beta9), Singapore, 2016-05-13: https://drive.google.com/file/d/0Bx3A13n3Q_EAa3NQYjBOWjhjZm8/view?usp=sharing
+.. _SCT course (v3.0_beta1), Montreal, 2016-04-19: https://drive.google.com/file/d/0Bx3A13n3Q_EAenltM2ZvZUNEdjQ/view?usp=sharing
+.. _SCT Hands-on Workshop (v2.0.4), Toronto, 2015-06-15: https://www.dropbox.com/s/f9887yrbkcfujn9/sct_handsOn_20150605.pdf?dl=0
+
+If you search for more specific knowledge, you could find some answers in the :ref:`tutorial section<tutorials>`.
```

Comment:
Slight rewording, since the tutorials are more-or-less the same as the SCT course content

```suggestion
If you would prefer to learn from webpage tutorials instead, then SCT also provides this course material in the form of step-by-step :ref:`<tutorials>`.
```